web-app/pages/_app.tsx

```diff
@@ -1,8 +1,24 @@
+/* eslint-disable react/jsx-props-no-spreading */
 import '../styles/globals.scss';
 import type { AppProps } from 'next/app';
+import Script from 'next/script';
 
 function MyApp({ Component, pageProps }: AppProps) {
-  // eslint-disable-next-line react/jsx-props-no-spreading
-  return <Component {...pageProps} />;
+  return (
+    <>
+      <Script strategy="lazyOnload" src={`https://www.googletagmanager.com/gtag/js?id=${process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS}`} />
```

Comment:
For 2 reasons, let's move this to a const above the function. 1) because I think it's best to have all `process.env` plucked out to const high in the file to make it easy to see how this file depend on `.env`. 2) because you're using `process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS` twice.

Also, let's rename NEXT_PUBLIC_GOOGLE_ANALYTICS to NEXT_PUBLIC_GOOGLE_ANALYTICS_ID or whatever it is.

So then you'll be doing something like `const googleAnalyticsId = process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS_ID; // then also add a comment pointing to where you learned our ID and docs about how you knew what script to put below in this file`

Reply:
@ozanisgor Any progress on this? Thanks.

Reply:
@ryancwalsh my commit is ready but I am waiting for the `Tracking ID`. Asked that to Sherif and he direct me to Shashi. So last friday I wrote to Shashi but couldn't get the answer. I heard he is sick maybe this is the reason. So I didn't push. As soon as I get the `Tracking ID` I will push the code.